### DISCUSSION TITLE
Renovate open a MR with invalid pom.xml

### DISCUSSION THREAD

**Discussion by thiagofabre** [renovatebot/renovate#18538] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

33.2.2

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

15.4.2-ee

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

The Renovate identifies the correct dependency, but fail to add the changes in the PR.

git diff
![Screenshot from 2022-10-24 16-29-41](https://user-images.githubusercontent.com/116581167/197735933-ea2c1bb2-0042-4a88-a16b-4bac39c9c3f8.png)

Does anyone have any idea what this can happen?

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by viceice** [comment, marked as answer]
see https://github.com/renovatebot/renovate/discussions/18521
